src/build.rs

```diff
@@ -102,12 +103,9 @@ pub(crate) fn build_plugin(
         }
     }
 
-    let mut args = vec!["build", "--target=wasm32-wasi"];
-    // TODO: don't hard-code --release
```

Comment:
In order to support `--debug`, we'd have to enable debugging in the host environment: https://docs.wasmtime.dev/examples-rust-debugging.html

Removing it for now but also I'm not sure how viable it'd be to do this.